base/some.jl

```diff
@@ -40,6 +40,15 @@ Throw an error if `x === nothing`, and return `x` if not.
 notnothing(x::Any) = x
 notnothing(::Nothing) = throw(ArgumentError("nothing passed to notnothing"))
 
+"""
```

Comment:
Needs to be added to the documentation so it shows up.

Reply:
Done - I added it next to `Nothing`.